### PR TITLE
view all users, nav bar update to contain link to user managemnet

### PR DESCRIPTION
**Pull request by jesmeeker**
Closes Ticket #13

# Description 

Displays a list of all Users, sorted by username with username, first name, last name and email. 
Updated NavBar to include User Management link

---

# Steps to Test

1. Fetch the [jes-view-user-list](https://github.com/nss-day-cohort-60/rare-client-tenacious-tators/compare/main...jes-view-user-list?expand=1) branch and switch to it
2. open Rare-Server and run server
3. npm start
4. login with username and password stored in your SQL db
5. click on User Management on navbar
6. verify that your users are displaying in Ascending order by username

### CODE REVIEW

File: src/components/users/User.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// import "./Animals.css"
```
Comment (stevebrownlee) [inline src/components/users/User.js:1]: No commented out code should go into the main branch

File: src/components/users/UserList.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+import { useNavigate } from 'react-router-dom'
+import { getUsers, getUsersBySearchTerm } from "../../managers/users"
+import { User } from "./User"
+import "./Users.css"
+// import { UserSearch } from "./UserSearch"
```
Comment (stevebrownlee) [inline src/components/users/UserList.js:6]: No commented out code should go into the main branch

File: src/components/users/UserList.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+export const UserList = () => {
+
+  const [users, setUsers] = useState([])
+//   const [searchTerm, setSearchTerm] = useState('')
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    getUsers().then(usersData => setUsers(usersData))
+  }, [])
+//   useEffect(() => {
```
Comment (stevebrownlee) [inline src/components/users/UserList.js:17]: No commented out code should go into the main branch

**Review by stevebrownlee — CHANGES_REQUESTED** [review]
Remove all commented out code

File: src/components/users/UserList.js (hunk 2 above)
Comment (jesmeeker) [reply src/components/users/UserList.js]: removed.

File: src/components/users/UserList.js (hunk 3 above)
Comment (jesmeeker) [reply src/components/users/UserList.js]: removed

File: src/components/users/User.js (hunk 1 above)
Comment (jesmeeker) [reply src/components/users/User.js]: removed

**Comment by lauren-hanson** [comment]
Everything looks lovely & ready to merge!

**Review by lauren-hanson — APPROVED** [review]
Everything looks lovely & ready to merge!

**Pull request merged**